### PR TITLE
Updating code of conduct translation: Hindi

### PR DESCRIPTION
**Pull request by nate-double-u**
Updating Hindi translation.

The CNCF has had a professional translation service provide this version of the code of conduct.

Deploy preview: https://github.com/nate-double-u/cncf-foundation/blob/2024-03-25-NW-code-of-conduct-translation-update-Hindi/code-of-conduct-languages/hi.md

Note: I didn't do the translation on this (just providing the markdown), but comments are welcome!

### CODE REVIEW

File: code-of-conduct-languages/hi.md [hunk 1]
```diff
@@ -1,99 +1,66 @@
...
-मुद्दों की रिपोर्टिंग(issues), सुविधा अनुरोध पोस्ट करने, दस्तावेज़ीकरण (documentation) अद्यतन करने के माध्यम से,
-पुल अनुरोध(PR) या पैच सबमिट करना, सम्मेलनों या कार्यक्रमों में भाग लेना, या अन्य समुदाय या परियोजना गतिविधियों में शामिल होना।
-
-हम उम्र, शरीर के आकार, जाति, विकलांगता, जातीयता, अनुभव के स्तर, पारिवारिक स्थिति, लिंग, लिंग पहचान और अभिव्यक्ति, वैवाहिक स्थिति, सैन्य या अनुभवी स्थिति, राष्ट्रीयता, व्यक्तिगत उपस्थिति, जाति, धर्म, यौन अभिविन्यास, सामाजिक आर्थिक स्थिति, जनजाति, या विविधता के किसी भी अन्य आयाम की परवाह किए बिना सीएनसीएफ समुदाय में भागीदारी को हर किसी के लिए उत्पीड़न-मुक्त अनुभव बनाने के लिए प्रतिबद्ध हैं।
+## सामुदायिक आचार संहिता
+
+CNCF समुदाय में योगदानकर्ताओं (Contributors), अनुरक्षकों(Maintainers) और प्रतिभागियों (Participants) के रूप में, और एक खुले और स्वागत करने वाले समुदाय को बढ़ावा देने के हित में, हम उन सभी लोगों का सम्मान करने की प्रतिज्ञा करते हैं जो मुद्दों की रिपोर्टिंग (issues) करने, सुविधा अनुरोध पोस्ट करने, दस्तावेज़ीकरण (documentation) अद्यतन करने, पुल अनुरोध या पैच सबमिट करने, सम्मेलनों या कार्यक्रमों में भाग लेने, या अन्य समुदाय या परियोजना गतिविधियों में शामिल होने के माध्यम से भाग लेते हैं या योगदान करते हैं।
```
Comment (SubhasmitaSw) [inline code-of-conduct-languages/hi.md:3]: Small nit It'll be more authentic if we replace submit(सबमिट) to जामा

File: code-of-conduct-languages/hi.md [hunk 2]
```diff
@@ -1,99 +1,66 @@
...
 ## दायरा
 
 यह आचार संहिता लागू होती है:
 * परियोजना और सामुदायिक स्थानों के भीतर,
-* अन्य स्थानों पर जब एक व्यक्तिगत सीएनसीएफ समुदाय प्रतिभागी के शब्द या कार्य सीएनसीएफ परियोजना, सीएनसीएफ समुदाय, या किसी अन्य सीएनसीएफ समुदाय प्रतिभागी पर निर्देशित होते हैं या उनके बारे में 
-  होते हैं।
+* अन्य स्थानों पर जब CNCF समुदाय के किसी व्यक्तिगत प्रतिभागी के शब्द या कार्य CNCF परियोजना, CNCF समुदाय, या CNCF समुदाय के किसी अन्य प्रतिभागी पर निर्देशित होते हैं या उसके बारे में होंं।
 
-### CNCF घटनाएँ
+## CNCF इवेंट्स
```
Comment (SubhasmitaSw) [inline code-of-conduct-languages/hi.md:13]: replace इवेंट्स with आयोजन

File: code-of-conduct-languages/hi.md [hunk 3]
```diff
@@ -1,99 +1,66 @@
...
-
-* कामुक भाषा या कल्पना का प्रयोग
-* ट्रोलिंग, अपमानजनक या अपमानजनक टिप्पणियाँ, और व्यक्तिगत या राजनीतिक हमले
+* रचनात्मक फीडबैक देना और शालीनता से स्वीकार करना
+* जिम्मेदारी स्वीकार करना और अपनी गलतियों से प्रभावित हुए लोगों से माफी मांगना, और ऐसे अनुभव से सीखना
+* सिर्फ हमारे लिए व्यक्तिगत रूप से नहीं, बल्कि पूरे समुदाय के हित के लिए क्या बेहतर है उस पर ध्यान केंद्रित करना
+* स्वागतपूर्ण और समावेशी भाषा का उपयोग करना
+
+अस्वीकार्य व्यवहार के उदाहरण निम्नलिखित हैं, लेकिन यहीं तक सीमित नहीं हैं:
+* कामुक भाषा या छवियों का उपयोगा
```
Comment (SubhasmitaSw) [inline code-of-conduct-languages/hi.md:30]: correct spelling for उपयोगा, replace with उपयोग

File: code-of-conduct-languages/hi.md [hunk 4]
```diff
@@ -1,99 +1,66 @@
...
+* ट्रोलिंग, अपमानजनक या निन्दात्मक टिप्पणियाँ, और व्यक्तिगत या राजनीतिक हमले
 * किसी भी रूप में सार्वजनिक या निजी उत्पीड़न
-* दूसरों की निजी जानकारी, जैसे भौतिक या ईमेल, प्रकाशित करना पता, उनकी स्पष्ट अनुमति के बिना
+* दूसरों की स्पष्ट अनुमति के बिना उनकी निजी जानकारी, जैसे भौतिक या ईमेल पता, प्रकाशित करना
 * हिंसा, हिंसा की धमकी देना, या दूसरों को हिंसक व्यवहार में शामिल होने के लिए प्रोत्साहित करना
-* किसी की सहमति के बिना उसका पीछा करना या पीछा करना अवांछनीय शारीरिक संपर्क
-* अवांछित यौन या रोमांटिक ध्यान या प्रगति
-* अन्य आचरण जिसे उचित रूप से अनुचित माना जा सकता है पेशेवर रूप से 
-  
+* बिना सहमति के किसी का पीछा करना या फॉलो करना।
```
Comment (SubhasmitaSw) [inline code-of-conduct-languages/hi.md:35]: पीछा करना या फॉलो -- they both are same, either पीछा or फॉलो should be used

File: code-of-conduct-languages/hi.md [hunk 5]
```diff
@@ -1,99 +1,66 @@
...
+* अवांछित शारीरिक संपर्क
+* सेक्शुअल या रोमांटिक अवांछित ध्यान या इशारे
```
Comment (SubhasmitaSw) [inline code-of-conduct-languages/hi.md:37]: no need for रोमांटिक word, सेक्शुअल या अवांछित ध्यान या इशारे is fine

File: code-of-conduct-languages/hi.md [hunk 6]
```diff
@@ -1,99 +1,66 @@
...
+* अन्य आचरण जिसे एक प्रोफेशनल वातावरण में उचित रूप से अनुचित माना जा सकता है
+
 निम्नलिखित व्यवहार भी निषिद्ध हैं:
 * आचार संहिता की जांच के संबंध में जानबूझकर गलत या भ्रामक जानकारी प्रदान करना या किसी जांच में जानबूझकर छेड़छाड़ करना।
-* किसी व्यक्ति के विरुद्ध प्रतिशोध लेना क्योंकि उन्होंने किसी घटना की सूचना दी थी या गवाह के रूप में किसी घटना के बारे में जानकारी प्रदान की थी।
+* किसी व्यक्ति के खिलाफ प्रतिशोध लेना क्योंकि उन्होंने किसी घटना की रिपोर्ट की थी या गवाह के रूप में किसी घटना के बारे में जानकारी प्रदान की थी।
```
Comment (SubhasmitaSw) [inline code-of-conduct-languages/hi.md:42]: why we replaced सूचना with रिपोर्ट here?

File: code-of-conduct-languages/hi.md [hunk 7]
```diff
@@ -1,99 +1,66 @@
...
+परियोजना अनुरक्षकों के पास इस आचार संहिता के अनुरूप नहीं होने वाली टिप्पणियों, प्रतिबद्धताओं, कोड, विकी संपादनों, मुद्दों और अन्य योगदानों को हटाने, संपादित करने या अस्वीकार करने का अधिकार और जिम्मेदारी है। इस आचार संहिता को अपनाकर, परियोजना अनुरक्षक CNCF परियोजना के प्रबंधन के हर पहलू पर इन सिद्धांतों को निष्पक्ष रूप से और लगातार लागू करने के लिए प्रतिबद्ध हैं। जो प्रोजेक्ट अनुरक्षक आचार संहिता का पालन नहीं करते या लागू नहीं करते, उन्हें प्रोजेक्ट टीम से अस्थायी या स्थायी रूप से हटाया जा सकता है।
 
 ## रिपोर्टिंग
 
-कुबेरनेट्स समुदाय में होने वाली घटनाओं के लिए, <conduct@kubernetes.io> के माध्यम से [कुबेरनेट्स आचार संहिता समिति](https://git.k8s.io/community/committee-code-of-conduct) से संपर्क करें। आप तीन व्यावसायिक दिनों के भीतर प्रतिक्रिया की उम्मीद कर सकते हैं।
+कुबेरनेट्स समुदाय में होने वाली घटनाओं के लिए, [conduct@kubernetes.io](mailto:conduct@kubernetes.io)<span style="text-decoration:underline;"> [कुबेरनेट्स आचार संहिता समिति](https://git.k8s.io/community/committee-code-of-conduct)</span> से संपर्क करें। आप तीन व्यावसायिक दिनों के भीतर जवाब प्राप्त करने की उम्मीद कर सकते हैं।
```
Comment (SubhasmitaSw) [inline code-of-conduct-languages/hi.md:48]: <span style="text-decoration:underline;"> --> needs to be removed

File: code-of-conduct-languages/hi.md [hunk 8]
```diff
@@ -1,99 +1,66 @@
...
+अन्य परियोजनाओं के लिए, या ऐसी घटनाओं के लिए जो परियोजना-अज्ञेयवादी हैं या कई CNCF परियोजनाओं को प्रभावित करती हैं, कृपया [conduct@cncf.io](mailto:conduct@cncf.io) पर [CNCF आचार संहिता समिति](https://www.cncf.io/conduct/committee/) से संपर्क करें<span style="text-decoration:underline;">।</span> वैकल्पिक रूप से, आप अपनी रिपोर्ट सबमिट करने के लिए [CNCF आचार संहिता समिति](https://www.cncf.io/conduct/committee/) के किसी भी व्यक्तिगत सदस्य से संपर्क कर सकते हैं। रिपोर्ट सबमिट करने के तरीके के बारे में अधिक विस्तृत निर्देशों के लिए, जिसमें गुमनाम रूप से रिपोर्ट सबमिट करने का तरीका भी शामिल है, कृपया हमार [घटना समाधान प्रक्रियाएँ](https://github.com/cncf/foundation/blob/main/code-of-conduct/coc-incident-resolution-procedures.md) देखें। आप तीन व्यावसायिक दिनों के भीतर जवाब प्राप्त करने की उम्मीद कर सकते हैं।
```
Comment (SubhasmitaSw) [inline code-of-conduct-languages/hi.md:50]: correct spelling for हमार to हमारे

File: code-of-conduct-languages/hi.md [hunk 9]
```diff
@@ -1,99 +1,66 @@
...
 
-Linux Foundation द्वारा निर्मित CNCF इवेंट में होने वाली घटनाओं के लिए कृपया संपर्क करें eventconduct@cncf.io.
+Linux Foundation द्वारा निर्मित CNCF इवेंट में होने वाली घटनाओं के लिए, कृपया [eventconduct@cncf.io](mailto:conduct@cncf.io) पर संपर्क करें।
```
Comment (SubhasmitaSw) [inline code-of-conduct-languages/hi.md:52]: translation needed here for लिनक्स फाउंडेशन (Linux Foundation), I guess we can also write like this to keep both सीएनसीएफ (CNCF)

Comment (SubhasmitaSw) [reply code-of-conduct-languages/hi.md:52]: पूर्ण वीरान (Full Stop) is missing in most of the ending statements

**Comment by nate-double-u** [comment]
Thanks so much for the review @SubhasmitaSw!

Could you update your suggestions to use the GitHub's ` ```suggestion` fence format (see step 6 in [Adding comments to a pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#adding-comments-to-a-pull-request))? This will make it easier for me to commit your suggestions since I don't read or write Hindi.

Also, could you wrangle another Hindi localization reviewer to confirm your suggestions? I'm happy to make changes, but since I don't read Hindi, I'd like at least a second review before accepting.

**Comment by nate-double-u** [comment]
Hi @SubhasmitaSw, just following up. Are you able to find someone else to help confirm your suggestions for me?

**Comment by nate-double-u** [comment]
Hi @SubhasmitaSw, I'm going to merge this as is. I invite you to open a follow up PR with your suggested changes.

**Pull request merged**